packages/patrol/lib/src/platform/selector.dart

```diff
@@ -96,6 +96,37 @@ class Selector implements CompoundSelector {
       web_selector.WebSelector(text: text, cssOrXpath: className);
 }
 
+/// A compound selector that behaves like NativeSelector, allowing platform-specific
+/// iOS and Android selectors to be specified separately.
```

Comment:
>iOS and Android **and Web**

:) Technically we could future proof it and say just `allowing platform-specific selectors to be specified separately`